content/en/apps/examples/supervisor-reference-app.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Supervisor Reference Application"
+linkTitle: "Supervisor Reference Application"
+weight:
+description: >
+  Supervisor Reference Application
+relatedContent: >
+  apps/concepts
+  apps/reference/forms/app
+  apps/reference/tasks
+---
+
+{{% pageinfo %}}
+
+A reference app for CHW supervisors to support performance management of CHWs using a mobile app.
+
+Medic has worked with D-tree to build a CHT supervisor reference app, the app is built to support community health workers (CHWs) supervisors to continually monitor and improve the program quality for Zanzibar National community health program Jamii ni Afya. The supervisor reference application is designed to enable supervisors access CHW performance information and any other information required to supervise CHWs, mentor CHWs and support CHWs to provide quality community health services. This reference app provides an example that CHT App Developers can easily customize to meet the needs for their specific program areas to support CHW program management.
```

Comment:
We should use full org names and links the first time we reference Medic and D-tree. Here is a suggested version, with other minor edits.

```suggestion
[Medic](https://medic.org/) has worked with [D-tree International](https://www.d-tree.org/) to build a CHT supervisor reference app. The app supports community health worker (CHW) supervisors to continually monitor and improve the program quality for Zanzibar National community health program Jamii ni Afya. The supervisor reference application is designed to enable supervisors to access CHW performance information and any other information required to supervise, mentor, and support CHWs to provide quality community health services. This reference app provides an example that CHT app developers can easily customize to meet the needs for their specific program areas to support CHW program management.
```